developer_manual/design/navigation.rst

```diff
@@ -2,10 +2,61 @@
 .. codeauthor:: John Molakvoæ <skjnldsv@protonmail.com>
 ..  _navigation:
 
+..  _newbutton:
+
+===============
+Introduction
+===============
+
+The navigation section of any nextcloud app is the left sidebar.
+It is basically composed of
+
+* a 'new' button
+* a menu
+* a settings area
+
+The 'new' button and the settings area are optional.
```

Comment:
Should we add a global code block to show a full example?

Reply:
Should be clear from the other examples imo.